.github/workflows/lintr.yml

```diff
@@ -44,7 +44,9 @@ jobs:
           wget -O .lintr https://raw.githubusercontent.com/Boehringer-Ingelheim/dv.templates/main/.lintr
 
       - name: Lintr 🔍
-        run: lintr::lint_package()
+        run: |
+          devtools::load_all()
```

Comment:
I would rather add 
`object_usage_linter = NULL`
to .lintr file than do acrobatics with rcmdcheck and devtools combined

Reply:
Good for me